-lib-/logic.rb

```diff
@@ -0,0 +1,79 @@
+class Player
+    attr_accessor :character, :name
```

Comment:
Use attr_readers/attr_writers instead of atrr_acessors.
make this change where necessary